<launch>
  <node machine="master" pkg="tf" type="static_transform_publisher" name="odometry_frame_broadcaster" args="10 10 0   3.93 0.0 0.0 /world /odometry_frame 100" />

  <node machine="master" pkg="pioneer_tools" type="odometry_frame_publisher" name="odometry_convertor" args="odometry:=/RosAria/pose" >
    <param name="child_frame" value="/robot_pos" type="string" />
    <param name="offset_x" value="5." type="double" />
    <param name="offset_y" value="0." type="double" />
    <param name="offset_z" value="0." type="double" />

  </node>

  <node machine="master" pkg="tf" type="static_transform_publisher" name="robot_frame_broadcaster" args="0 0 0 0 0 0 /robot_pos /sonar_frame 100" />
</launch>